langchain_falkordb/message_history.py

```
@@ -117,8 +117,7 @@ class with the session and connection details.
                 f"{self._node_label}", "id", dim=5, similarity_function="cosine"
             )
         except Exception as e:
-            if "already indexed" in str(e):
-                raise ValueError(f"{self._node_label} has already been indexed")
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)